docs/DXIL.rst

```diff
@@ -267,6 +267,17 @@ The attribute 'waveops-include-helper-lanes' is utilized to indicate that wave o
 
 'fp32-denorm-mode' is employed to define the denorm mode for the function. The possible values for this attribute can be 'any', 'preserve', or 'ftz'.
 
+Module Flags
+------------
+
+DXIL uses `Module Flags Metadata <https://releases.llvm.org/3.7.1/docs/LangRef.html#module-flags-metadata>`_ to convey additional information about the module.
+
+The only module flags allowed in DXIL are "Dwarf Version", "Debug Info Version".
```

Comment:
What does the validator verify for these flags? Does it verify just the name, or does it rely on them having a specific structure?

Reply:
We didn't check it at all in validation currently :(
Since we don't modify the Dwarf Version and Debug Info Version in dxc, shall we limit Dwarf Version always be 4 and Debug Info Version always be 3?

Reply:
What _does_ the validator check? If it doesn't check anything why are we claiming that these are the only valid flags?

Reply:
Because those flags are only flags dxc will generated.